testsuite/gateway/__init__.py

```diff
@@ -177,6 +177,44 @@ def remove_all_backend(self):
         """Sets match for a specific backend"""
 
 
+@dataclass
+class GatewayListener:
+    """
+    Dataclass of Gateway listener object.
+    When used in `add_listener()` function you MUST specify a unique name!
```

Comment:
Is this line still needed here and in `TLSGatewayListener`?

Reply:
Yes this warning is still needed as listeners in Gateway must have unique names. I do not generate random names at runtime, this is responsibility of the user if they want to add another listener.